### DISCUSSION TITLE
Quantity in shopping cart not shown but visible on the cart badge

### DISCUSSION THREAD

**Discussion by geeekgod** [solidusio/solidus#5447] [New Admin UI/UX]
## Potential bug where the quantities are not being displayed

When I add a product to the cart with more than 10 quantity. In the shopping cart I'm seeing the quantity as `0` which in this case should have been the actual quantity of the particular product.

### Products quantity not being showed here:
<img width="658" alt="image" src="https://github.com/solidusio/solidus/assets/56874272/4dfbf62e-31b7-41dc-acc2-60b66482e005">

### The total quantity being rendered in the badge here:

<img width="658" alt="image" src="https://github.com/solidusio/solidus/assets/56874272/80d7ccc6-cd86-49f9-93bd-1cae0d879064">

**Comment by kennyadsl** [comment]
Thanks for reporting, is this happening on the demo only?
